mne_g3d/viz.py

```diff
@@ -65,27 +73,68 @@ def plot_brain_mesh(rh_vertices=None,
 
     fig = ipv.figure(width=fig_size[0], height=fig_size[1], lighting=True)
 
+    if act_data is not None:
+        cmap = get_mesh_cmap(act_data, cmap_str=cmap_str)
+        # data mapping into [0, 1] interval
+        arr_act_data = np.concatenate(tuple(act_data.values()))
```

Comment:
same thing here about `dict` order

this is the only thing really done in `get_mesh_cmap`, too, so it's another argument for not having that function